_config.yml

```diff
@@ -25,34 +25,7 @@ kramdown:
 # Site owner
 owner:
   name: Bitcoin Core
-  avatar: bio-photo.jpg
```

Comment:
Asset unused after this?

Reply:
Not yet. Still used in _includes/author-bio.html

I'll remove that in a follow-up.